tests/utils/namespace.go

```diff
@@ -175,38 +51,23 @@ func (env TestingEnvironment) GetOperatorLogs(buf *bytes.Buffer) error {
 	return streamPodLog.Stream(env.Ctx, buf)
 }
 
-// DumpNamespaceOperatorLogs writes the operator logs related to a namespace into a writer
-// and also into a file
-func (env TestingEnvironment) DumpNamespaceOperatorLogs(namespace, testName string, output io.Writer) {
```

Comment:
IMO this is a loss. This code was ensuring we saved the operator logs for the test's namespace only. And would also show on Ginkgo operator logs for this namespace that were Error or Warn.
Is this functionality captured in a different way in this PR?

Reply:
I have no problem with the general operator log tailing being converted to stern.
It's the namespace-filtered operator logs that I think were an improvement over a single file with all the operator logs.

Reply:
@jsilvela after months of having this I haven't found any one using the operator logs in the Ginkgo namespace, instead, is more noise than help in there because start to fill up the Ginkgo buffer and one start to miss the focus on the error, about the namespace captured log, that's right, we may improve that in the future but for now an operator log that you can grep with the namespace looks fair enough, this is a long process of improving the e2e tests, for sure if we found needed we can incorporate that functionality in the next few improvements but for now I think it's ok to keep this out, there's other PRs that I'll be opening to improve the e2e

Reply:
> after months of having this I haven't found any one using the operator logs in the Ginkgo namespace
not been months.
This was merged 20 days ago. https://github.com/cloudnative-pg/cloudnative-pg/pull/5215
As to "not found anyone" -- right, so, I did use it, it helped for an investigation, which is why I decided to make the functionality available.

> we may improve that in the future but for now an operator log that you can grep with the namespace looks fair enough
This is not about adding code, but about deleting less of it.

that said, @sxd , I'm not going to ask for changes, go ahead. But I'm also not giving approve.